### ISSUE TITLE
Dettection of trees using custom Dataset

### ISSUE THREAD

**Issue by nabeelyousfi** [weecology/DeepForest#26]
Hey

Thank you for your nice work. I want to detect trees in the given image. Can you please tell me how can i do this with your code.There is no extensive documentation is provided.

![frame0](https://user-images.githubusercontent.com/16258518/69538138-ebd4a180-0f81-11ea-89ab-2d63d1abfc45.jpg)

**Comment by bw4sz** [comment]
Thanks for your interest. DeepForest is in pre-beta at 0.0.7 on dev branch. 
Have you tried what's on the README? What errors do you receive? It will be useful to have a first test of dependencies install. 

```
from deepforest import deepforest
from deepforest import utilities

#Download latest model release from github
utilities.use_release()    

#Load model class with release weights
test_model = deepforest.deepforest(weights="data/universal_model_july30.h5")

#predict image
image = test_model.predict_image(image_path = "tests/data/OSBS_029.tif")
```

Once we get through that, we can try either ```predict_image``` or ```predict_tile`` depending on whether your image is too large to fit into memory.

more documentation will appear: https://deepforest.readthedocs.io/en/dev/, again the repo is not ready for public consumption.

**Comment by nabeelyousfi** [comment]
Thank you for you response as I run all the dependencies and also give this image for predict so these errors are occurred.
![Screenshot from 2019-11-26 13-41-51](https://user-images.githubusercontent.com/16258518/69609102-12014c80-1029-11ea-8fe5-a37d8aff87ea.png)

**Comment by bw4sz** [comment]
Those errors do not look like DeepForest errors, but that you do not have a functioning tensorflow version. Let's check that. Here is the standard test from keras.

https://keras.io/examples/mnist_cnn/

copy and paste into a file and run:

```
(DeepForest) MacBook-Pro:Downloads ben$ python test.py
Using TensorFlow backend.
Downloading data from https://s3.amazonaws.com/img-datasets/mnist.npz
11493376/11490434 [==============================] - 6s 0us/step
x_train shape: (60000, 28, 28, 1)
60000 train samples
10000 test samples

Train on 60000 samples, validate on 10000 samples
Epoch 1/12
60000/60000 [==============================] - 71s 1ms/step - loss: 0.2563 - accuracy: 0.9217 - val_loss: 0.0537 - val_accuracy: 0.9824
Epoch 2/12
60000/60000 [==============================] - 65s 1ms/step - loss: 0.0857 - accuracy: 0.9740 - val_loss: 0.0377 - val_accuracy: 0.9868
Epoch 3/12
60000/60000 [==============================] - 64s 1ms/step - loss: 0.0633 - accuracy: 0.9813 - val_loss: 0.0380 - val_accuracy: 0.9871
Epoch 4/12
60000/60000 [==============================] - 71s 1ms/step - loss: 0.0541 - accuracy: 0.9841 - val_loss: 0.0284 - val_accuracy: 0.9904
Epoch 5/12
60000/60000 [==============================] - 67s 1ms/step - loss: 0.0444 - accuracy: 0.9870 - val_loss: 0.0332 - val_accuracy: 0.9892
Epoch 6/12
60000/60000 [==============================] - 64s 1ms/step - loss: 0.0408 - accuracy: 0.9871 - val_loss: 0.0291 - val_accuracy: 0.9897
Epoch 7/12
60000/60000 [==============================] - 64s 1ms/step - loss: 0.0362 - accuracy: 0.9889 - val_loss: 0.0246 - val_accuracy: 0.9918
Epoch 8/12
60000/60000 [==============================] - 66s 1ms/step - loss: 0.0338 - accuracy: 0.9898 - val_loss: 0.0267 - val_accuracy: 0.9908
Epoch 9/12
60000/60000 [==============================] - 63s 1ms/step - loss: 0.0314 - accuracy: 0.9906 - val_loss: 0.0261 - val_accuracy: 0.9922
Epoch 10/12
60000/60000 [==============================] - 74s 1ms/step - loss: 0.0302 - accuracy: 0.9907 - val_loss: 0.0279 - val_accuracy: 0.9921
Epoch 11/12
60000/60000 [==============================] - 72s 1ms/step - loss: 0.0276 - accuracy: 0.9917 - val_loss: 0.0276 - val_accuracy: 0.9911
Epoch 12/12
60000/60000 [==============================] - 67s 1ms/step - loss: 0.0261 - accuracy: 0.9921 - val_loss: 0.0271 - val_accuracy: 0.9909
Test loss: 0.027092657606024296
Test accuracy: 0.9908999800682068
```

that will help identify if this is a global error.

**Comment by bw4sz** [comment]
closing. Welcome to reopen if you can show you have a function tensorflow version.

New docs have been published and checked. See

https://deepforest.readthedocs.io
